src/encoded/static/components/matrix_chip_seq.js

```diff
@@ -118,50 +117,37 @@ const getChIPSeqData = (context, assayTitle, organismName) => {
     };
 };
 
-
-/**
- * Determines if the matrix update is large
- *
- * @param {*} currentChIPSeqData
- * @param {*} newChIPSeqData
- * @returns
- * Note- This is hacky and needs to be reconsidered
- */
-const isMatrixUpdateLarge = (currentChIPSeqData, newChIPSeqData) => {
-    const isCurrentMatrixLarge = currentChIPSeqData && currentChIPSeqData.dataRow && (currentChIPSeqData.dataRow.length > LARGE_MATRIX_MIN_SIZE);
-    const isNewMatrixLarge = newChIPSeqData && newChIPSeqData.dataRow && (newChIPSeqData.dataRow.length > LARGE_MATRIX_MIN_SIZE);
-
-    return isCurrentMatrixLarge || isNewMatrixLarge;
-};
-
 /**
  * Transform chIP Seq data to a form DataTable-object can understand.
  *
  * @param {chIPSeqData} chIPSeqData
  * @param {string} selectedTabLevel3 - Sub tab to use
  * @returns {object} DataTable-ready structure.
  */
-const convertTargetDataToDataTable = (chIPSeqData, selectedTabLevel3) => {
+const convertTargetDataToTable = (chIPSeqData, selectedTabLevel3) => {
     if (!chIPSeqData || !chIPSeqData.headerRow || !chIPSeqData.dataRow) {
-        return {
-            rows: [],
-            rowKeys: [],
-            tableCss: 'matrix',
-        };
+        return [];
     }
 
     // add assay_title = Mint chip-seq if the assay selected in Histone chip-seq
     const isAssayTitleHistone = chIPSeqData.assayTitle === 'Histone ChIP-seq';
-
-    const dataTable = [];
-    const headerRow = chIPSeqData.headerRow.map((x) => ({
-        header: <a href={`/search/?type=Experiment&status=released&replicates.library.biosample.donor.organism.scientific_name=${chIPSeqData.organismName}&biosample_ontology.term_name=${x}&assay_title=${chIPSeqData.assayTitle}${isAssayTitleHistone ? '&assay_title=Mint-ChIP-seq' : ''}`} title={x}>{x}</a>,
-    }));
-
-    dataTable.push({
-        rowContent: [{ header: null }, ...headerRow],
-        css: 'matrix__col-category-header',
-    });
+    const removeSpecialCharacters = (name) => (!name ? name : name.replace(/\s/g, ''));
```

Comment:
I feel like we should add a function to globals.js to convert arbitrary strings to something acceptable for ids and maybe CSS class names.